golang/go-cloud-run-hello-world/README.md

```diff
@@ -0,0 +1,53 @@
+# Cloud Run Hello World with Cloud Code
+
+This "Hello World" is a [Cloud Run](https://cloud.google.com/run/docs) service that renders a webpage.
+
+----
+## Table of Contents
+
+### Cloud Code for Visual Studio Code
+
+1. [Getting Started](#getting-started])
+2. [What's in the box](https://cloud.google.com/code/docs/vscode/quickstart#whats_in_the_box)
+3. [Using the Command Line](#using-the-command-line)
```

Comment:
We shouldn't have the section of using command line (even though it should work when invoking appropriate command line). This is because we optimize for Cloud Code IDE experience (IntelliJ and vscode).

Reply:
We probably need two more sections:

1) Running locally
2) Running on Cloud Run

Reply:
Also, maybe "Next steps" with more useful links.

Reply:
Alright, will iterate this. However, I don't think it's good to hide the command-line operations. It might make sense for a simple example like this, but once it's more complicated understanding the escape valve is important.

Reply:
Mostly done. I don't think all the useful links exist yet.